CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 * [#6766](https://github.com/rubocop-hq/rubocop/pull/6766): Drop support for Ruby 2.2.0 and 2.2.1. ([@pocke][])
 * [#6733](https://github.com/rubocop-hq/rubocop/pull/6733): Warn duplicated keys in `.rubocop.yml`. ([@pocke][])
+* [#6769](https://github.com/rubocop-hq/rubocop/pull/6769): Replace md5 with sha1 for FIPS compliance. ([@dirtyharrycallahan][])
```

Comment:
RuboCop 0.65.0 has been released. Please rebase with the latest master branch.

Reply:
And CI has failed.

```console
  1) RuboCop Project changelog has link definitions for all implicit links
     Failure/Error:
       expect(changelog.include?("[#{name}]: http"))
         .to be(true), "CHANGELOG.md is missing a link for #{name}. " \
                       'Please add this link to the bottom of the file.'
```

https://circleci.com/gh/rubocop-hq/rubocop/37727

Please add a link definition for the link to the bottom of the changelog as `[@dirtyharrycallahan]: https://github.com/dirtyharrycallahan`.

```diff
+[@dirtyharrycallahan]: https://github.com/dirtyharrycallahan
```